pages/get-started.vue

```diff
@@ -280,9 +280,17 @@ ul {
     }
     a {
       padding: $margin;
+      transition: 0.1s ease-in;
       &:hover {
-        transition: 0.1s ease;
-        background-color: rgba(0, 0, 0, 0.025);
+        background-color: rgba(231, 210, 249, 0.15);
```

Comment:
To avoid. If you can attach this color to one if your variables that would be nicer. It's hard to know what kind of color it is

Reply:
Fixed